### PR TITLE
Esc closes bookend

### LINKED ISSUE #11644: Pressing ESC while on the amp-story bookend should close the bookend
We do not currently support pressing ESC to close the bookend, but as a modal, this is the expected behavior.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #11644 

- Closes bookend on pressing `esc`

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

File: extensions/amp-story/0.1/system-layer.js [hunk 1]
```diff
@@ -222,6 +222,12 @@ export class SystemLayer {
         this.onUnmuteAudioClick_(e);
       }
     });
+
+    this.win_.addEventListener('keyup', e => {
+      if (!this.closeBookendBtn_.hasAttribute('hidden') && e.key == 'Escape') {
```
Comment (newmuis) [inline extensions/amp-story/0.1/system-layer.js]: I'm sure we have a constant for escape; see `../../../src/utils/key-codes`

File: extensions/amp-story/0.1/system-layer.js [hunk 2]
```diff
@@ -222,6 +222,12 @@ export class SystemLayer {
...
+        this.onCloseBookendClick_(e);
```
Comment (newmuis) [inline extensions/amp-story/0.1/system-layer.js]: nit: maybe rename this function to something generic to both click and keypress like `closeBookend`?  @alanorozco might have better ideas for naming

Comment (prateekbh) [reply extensions/amp-story/0.1/system-layer.js]: Done

File: extensions/amp-story/0.1/system-layer.js (hunk 1 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/system-layer.js]: Done

**Pull request merged**